src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let returnValue = initialValue;
+    let startIndex = 0;
+
+    if (typeof initialValue === 'undefined' || initialValue == null) {
```

Comment:
```suggestion
    if (initialValue === undefined || initialValue == null) {
```

Reply:
it would be shorter but it isn't necessary)

Reply:
thanks, fixed it)